tests/testutils/db.py

```diff
@@ -0,0 +1,60 @@
+"""db.py.
+
+Database level utility class for testing against timed_release model.
+"""
+
+from functools import wraps
+import sys
+
+from timed_release import config
+from timed_release.connectors.mysql import base_model
+from timed_release.connectors.mysql import rds_db_engine
+from timed_release.connectors.mysql import rds_db_session
+
+
+def create_timed_release_model_table():
+    """Create Model table."""
+    _exit_if_not_test_environment(rds_db_session())
+    base_model.metadata.create_all(rds_db_engine)
```

Comment:
This create_all creates a table for the model classes where base_model has been injected.